nuxeo-services/nuxeo-platform-web-common/src/main/java/org/nuxeo/ecm/platform/ui/web/auth/NuxeoAuthenticationFilter.java

```diff
@@ -873,6 +870,10 @@ protected boolean handleLogout(ServletRequest request, ServletResponse response,
         } catch (LoginException e) {
             log.error("Unable to logout " + e.getMessage());
         }
+
+        // invalidate Session !
+        service.invalidateSession(request);
+
```

Comment:
Yes this is worth a comment somewhere, maybe the ticket. Basically computing the logoutUrl may need to look up what was stored in the session at login time, so we mustn't invalidate the session too soon.